### DISCUSSION TITLE
Files with a file name of % cannot be uploaded by remotely-save

### DISCUSSION THREAD

**Discussion by kk-ssh** [remotely-save/remotely-save#206] [Q&A]
![image](https://user-images.githubusercontent.com/58595738/193239013-4ccf5f5e-6dcf-4b4a-a66e-4778b1cda1e0.png)
The % in the filename is necessary for me, is there a solution? Thanks

**Comment by fyears** [comment]
it's limited by the service, and nothing i can do here
